oml/models/vit/clip.py

```diff
@@ -0,0 +1,159 @@
+        ),
+        "openai_vitb32_224": (
+            "https://openaipublic.azureedge.net/clip/models/40d365715913c9da98579312b702a82c18be219cc2a73407c4526f58eba950af/ViT-B-32.pt",
+            "3ba34e387b24dfe590eeb1ae6a8a122b",
+        ),
+        "openai_vitl14_224": (
+            "https://openaipublic.azureedge.net/clip/models/b8cca3fd41ae0c99ba7e8951adf17d267cdb84cd88be6f7c2e0eca1737a03836/ViT-L-14.pt",
+            "096db1af569b284eb76b3881534822d9",
+        ),
+        "openai_vitl14_336": (
+            "https://openaipublic.azureedge.net/clip/models/3035c92b350959924f9f00213499208652fc7ea050643e8b385c2dac08641f02/ViT-L-14-336px.pt",
+            "b311058cae50cb10fbfa2a44231c9473",
+        ),
+        # checkpoints pretrained by SberbankAI
+        "sber_vitb16_224": (
+            "https://huggingface.co/sberbank-ai/ruclip-vit-base-patch16-224/resolve/main/pytorch_model.bin",
+            "7882e07674d78c674e33cb892a68bbfc",
+        ),
+        "sber_vitb32_224": (
+            "https://huggingface.co/sberbank-ai/ruclip-vit-base-patch32-224/resolve/main/pytorch_model.bin",
+            "e2c4dab46a3cfa608bdd762973e90d32",
+        ),
+        "sber_vitl14_224": (
+            "https://huggingface.co/sberbank-ai/ruclip-vit-large-patch14-224/resolve/main/pytorch_model.bin",
+            "9b4a1cd25d15bad4ffd2ba6e34b8a67c",
+        ),
+    }
+
+    def __init__(
+        self,
+        arch: str,
+        normalise_features: bool = True,
+        weights: Optional[str] = None,
+        strict_load: bool = True,
+    ):
+        """
+        Args:
+            weights: Path to weights or special key for pretrained ones or ``None`` for random initialization.
+             You can check available pretrained checkpoints in ``ViTCLIPExtractor.pretrained_models``.
+            arch: Might be one of ``vitb16_224``, ``vitb32_224``, ``vitl14_224``, ``vitl14_336``.
+            normalise_features: Set ``True`` to normalise output features
+            strict_load: Whether the weights needed to be loaded strictly. Doesn't work with OpenAI's models.
+        """
+
+        super().__init__()
+
+        self.normalize = normalise_features
+
+        cfg = get_vit_config_by_name(arch)
+        embed_dim = cfg["embed_dim"]
+        image_resolution = cfg["image_resolution"]
+        layers = cfg["layers"]
+        width = cfg["width"]
+        patch_size = cfg["patch_size"]
+        heads = cfg["heads"]
+
+        self.visual = VisionTransformer(
+            input_resolution=image_resolution,
+            patch_size=patch_size,
+            layers=layers,
+            width=width,
+            heads=heads,
+            output_dim=embed_dim,
+        )
+
+        if weights is None:
+            return
+        if weights in self.pretrained_models:
+            weights, md5 = self.pretrained_models[weights]
+            weights = download_checkpoint(weights, md5)
+
+        try:  # check if weights are jitted
```

Comment:
We already know which weights are jitted, so, instead of doing try/except, we can do 
```
JITTED_WEIGHTS = [name1, name2,...]
if weights in JITTED_WEIGHTS:
    do smth
```